roles/vault/tasks/shared/issue_cert.yml

```diff
@@ -70,7 +109,7 @@
 - name: issue_cert | Copy certificate serial to all hosts
   copy:
     content: "{{ hostvars[issue_cert_hosts|first]['issue_cert_result']['json']['data']['serial_number'] }}"
```

Comment:
Maybe this should not include hostvars[issue_cert_hosts|first] and just directly reference issue_cert_result